index.html

```diff
@@ -2433,34 +2433,30 @@ <h2>Production and Consumption</h2>
         </li>
       </ol>
 
-      <p class="note" title="Representation-specific entries">
-Note that representation-specific entries will only have special processing
-rules defined by a single <a>representation</a>. Consumers of a different
-<a>representation</a> are required to preserve these entries in the
-<a>DID document</a> <a
-href="#data-model">data model</a> using their generic type processing rules
-to enable lossless conversion of <a>representations</a>. Similarly, producers
-are required to treat entries containing representation-specific syntax using
-generic type processing rules when producing a <a>representation</a> for which
-the entry is not defined. <a>Representations</a> are required to define producer
-behavior for any such entries defined by the <a>representation</a>.
-      </p>
-
       <p>
 The requirements for all <a>conforming producers</a> are as follows:
       </p>
 
       <ol>
         <li>
-A <a>conforming producer</a> MUST serialize all entries in the <a
-href="#data-model">data model</a> that do not have explicit processing rules for
-the <a>representation</a> being produced using only the <a>representation</a>'s
-data type processing rules.
+A <a>conforming producer</a> MUST take a <a>DID document</a> <a
+href="#data-model">data model</a> and a representation-specific entries <a
+data-cite="INFRA#maps">map</a> as input into the <a>production</a> process.
+The <a>conforming producer</a> MAY accept additional options as input
+into the <a>production</a> process.
+        </li>
+        <li>
+A <a>conforming producer</a> MUST serialize all entries in the <a>DID
+document</a> <a href="#data-model">data model</a>, and the
+representation-specific entries <a data-cite="INFRA#maps">map</a>, that do not
+have explicit processing rules for the <a>representation</a> being produced
+using only the <a>representation</a>'s data type processing rules and
+return the serialization after the <a>production</a> process completes.
```

Comment:
After reading this over several times, I *think* I've figured out a small
set of changes that make it clearer after fewer readings, hopefully down
to one read for sufficient clarity to act on.
```suggestion
A <a>conforming producer</a> MUST serialize all entries in the <a>DID
document</a> <a href="#data-model">data model</a>, and the relevant
entries in the representation-specific entries <a
data-cite="INFRA#maps">map</a>, that do not have explicit processing
rules for the <a>representation</a> being produced, using only the
<a>representation</a>'s data type processing rules; and return the
serialization after the <a>production</a> process completes.
```